src/main/java/oof/command/Command.java

```diff
@@ -28,12 +29,14 @@
      *
      * @param semesterList Instance of SemesterList that stores Semester objects.
      * @param tasks        Instance of TaskList that stores Task objects.
+     * @param trackerList  Instance of TrackerList that stores Tracker objects.
      * @param ui           Instance of Ui that is responsible for visual feedback.
      * @param storage      Instance of Storage that enables the reading and writing of Task
      *                     objects to hard disk.
      * @throws OofException Catches invalid commands given by user.
      */
-    public abstract void execute(SemesterList semesterList, TaskList tasks, Ui ui, Storage storage) throws OofException;
+    public abstract void execute(SemesterList semesterList, TaskList tasks, TrackerList trackerList,
+                                 Ui ui, Storage storage) throws OofException;
```

Comment:
Why is TrackerList added to the argument parameter when it is not used by most, if not all, commands.

Reply:
This has now been rectified.